mne/utils/dataframe.py

```diff
@@ -36,7 +36,7 @@ def _scale_dataframe_data(inst, data, picks, scalings):
     return data
 
 
-def _convert_times(times, time_format, meas_date=None, first_time=0):
+def _convert_times(times, time_format, meas_date=None, first_time=0, drop_nano=False):
```

Comment:
Might require fixes elsewhere but worth it for readability I think
```suggestion
def _convert_times(times, time_format, *, meas_date=None, first_time=0, drop_nano=False):
```